azure-pipelines/publishing/github-release.yml

```diff
@@ -6,7 +6,12 @@ trigger:
 pr:
 - none
 
-pool: dotestlab
+#pool: 1es_hosted_pool_ubuntu_1804
```

Comment:
I think we are ready to switch this one also to the 1ES pool right?

Reply:
Yup!